docroot/themes/custom/dcp/js/main.js

```diff
@@ -26,4 +26,23 @@
     }
   };
 
+  Drupal.behaviors.addActiveLink = {
+    attach: function (context, settings) {
+      $( window ).resize(function() {
+        if(window.innerWidth > 992) {
+          $('body').css('overflow', 'auto');
+          $('.navigation-mobile').hide();
+        }
+      });
+      $('.region-header .mobile-menu.trigger-icon').on('click',() => {
```

Comment:
condition missing!! should be applied only for mob and tab no?

Reply:
@prashant114606  can you provide me which specific line you are referring to

Reply:
these are for hamburger menu right?
and those are only on mob + tab

Reply:
yes these are only for mob and tab